enhancements/installer/openstack-for-power-vc-ipi.md

```diff
@@ -0,0 +1,300 @@
+---
+title: openstack-for-power-vc-ipi
+authors:
+  - "@mjturek"
+reviewers:
+  - "@sherine-k"
+  - "@prb112"
+  - "@clnperez"
+  - "@Prashanth684"
+
+approvers:
+  - "@patrickdillon"
+ 
+api-approvers:
+ - "@tbd"
+
+creation-date: 2025-06-18
+last-updated: 2025-09-23
+status: implementable
+tracking-link:
+  - https://issues.redhat.com/browse/MULTIARCH-5358
+see-also:
+  - enhancements/installer/ibm-cloud-for-power-vs-ipi.md
+---
+
+# openstack-for-power-vc-ipi
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for tech preview and GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This document describes how the [OpenStack](https://github.com/kubernetes-sigs/cluster-api-provider-openstack) provider can be leveraged to provide infrastructure to deploy
+OpenShift on the [Power Virtualization Center](https://www.ibm.com/docs/en/powervc/2.1.1?topic=power-virtualization-center-apis) offering. Power Virtualization Center, or PowerVC, is an IBM
+offering that provides virtualization management for their on-premise datacenters. Because PowerVC
+is based on OpenStack, the new PowerVC platform heavily uses the same paths in the installer that
+the OpenStack platform uses. The goal of this enhancement is to add a new powervc platform to the OpenShift
+installer, that essentially serves as a shim provider of the existing openstack platform. This new
+platform will acount for the differences between powervc and openstack to enable deployment of OpenShift to a PowerVC environment.
+
+## Motivation
+
+- Current deployments of OpenShift on Power on premise are user-provisioned, agent based, or use the assisted installer.
+-- While effective, they require a lot of user intervention to deploy.
+-- Automation around these features is not supported.
+-- Scaling is more difficult when deploying via these methods.
+- Customers are searching for a simplified method of deploying OpenShift into their existing PowerVC environments.
+- IPI provides an opinionated and simplified deployment method.
+-- Because PowerVC is built on top of OpenStack, we can use the OpenStack IPI platform nearly as-is.
+- Clusters deployed with IPI are equipped with an easy way to scale after deployment.
```

Comment:
Note that it's possible to deploy the control plane with UPI but still use platform:openstack so that the worker nodes are created and scaled by the Machine API. Having an IPI installer is not a prerequisite for that.

Reply:
Good point... I'll add some clarity